Dockerfile

```diff
@@ -220,6 +224,10 @@ ENTRYPOINT ["buildkitd"]
 
 FROM binaries AS buildkit-darwin
 
+FROM binaries AS buildkit-freebsd
+COPY --link --from=binaries / /usr/bin/
```

Comment:
This line doesn't make much sense. You are copying from `binaries` on top of `binaries`

Reply:
That's a very nice catch! Fixed